nixos/tests/mycroft.nix

```diff
@@ -0,0 +1,377 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }:
+
+with lib;
+
+{
+  name = "mycroft";
+  meta.maintainers = teams.mycroft.members;
+
+  machine = { pkgs, ... }:
+  {
+    services.mycroft = {
+      enable = true;
+    };
+
+    environment.etc."mycroft/mycroft.conf".text = ''
```

Comment:
Yeah, I just pasted the whole config and toyed around with it, looking what it would take to get it to run. If we only need minimal configuration that's great, but we'd need to link a place where defaults can be looked up.